language/sygus.tex

```diff
@@ -3015,6 +3013,77 @@ \section{Examples}%
 \end{lstlisting}
 \end{example}
 
+
+\begin{example}[PBE with Oracle Function Symbols]
+Oracle function symbols are used to represent black-box functions within specifications. Consider a PBE problem where the user has an external oracle and wishes to synthesise a summary of the external oracle for a specific set of inputs. This can be encoded
```

Comment:
```suggestion
Oracle function symbols are used to represent black-box functions within specifications. Consider a PBE problem where the user has an external oracle and wishes to synthesize a summary of the external oracle for a specific set of inputs. This can be encoded
```